versioned_docs/version-22.04/releases/centreon-core.md

```diff
@@ -17,6 +17,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 22.04.6
+
+#### Bug fixes
+
+- [Authentication] Fixed user theme retrieval via OpenID Connect login
```

Comment:
[Configuration] Fixed export of RRDcached path in Centreon Broker configuration
[Core] Improved database storage to avoid blocking Broker when maximum values are reached